collector/impl/prometheus_metrics_collector.go

```diff
@@ -31,6 +31,9 @@ import (
 	"github.com/hawkular/hawkular-openshift-agent/http"
 	"github.com/hawkular/hawkular-openshift-agent/log"
 	"github.com/hawkular/hawkular-openshift-agent/prometheus"
+	"github.com/hawkular/hawkular-openshift-agent/util/expand"
+	"os"
+	"strings"
```

Comment:
convention in HOSA code is to put build-in imports at the top - so put "os" and "strings" in the top section (with "fmt", "time", etc).